src/client/kotlin/tech/thatgravyboat/skycubed/features/overlays/DialogueOverlay.kt

```diff
@@ -76,28 +106,32 @@ object DialogueOverlay : Overlay {
                         Displays.background(
                             BACKGROUND_COLOR,
                             OverlaysConfig.npc.overlayRadius.toFloat(),
-                            Displays.padding(
-                                5,
-                                Displays.text(Text.join(
-                                    Text.of("[Y]es") { this.color = TextColor.GREEN },
-                                    CommonText.SPACE,
-                                    Text.of("[N]o") { this.color = TextColor.RED }
-                                ))
-                            )
+                            yesNoDisplay,
                         )
                     )
                 } else {
                     reset()
                 }
             } else {
                 val (name, message) = queue.removeFirstOrNull() ?: return
+                val armorStands = (McLevel.self as LevelInvoker).callGetEntities().all.filterIsInstance<ArmorStand>()
```

Comment:
instead of getting all entities use McLevel.self.getEntityByClass and pass in ArmorStand::class.java and then an aabb of the block with a height of a few blocks above